src/templates/summary/index.ts

```diff
@@ -12,12 +13,18 @@ type GroupedProposals = Record<ProposalStatus, GroupedSpaces[]>;
 const MAX_SHORT_BODY_LENGTH = 150;
 
 export async function getGroupedProposals(addresses: string[], startDate: Date, endDate: Date) {
+  const { flaggedSpaces, flaggedProposals } = await getModerationList();
+
   const follows = await getFollows(addresses);
-  const proposals = await getProposals(
-    follows.map(follow => follow.space.id),
-    startDate,
-    endDate
-  );
+  const proposals = (
+    await getProposals(
+      follows
+        .map(follow => follow.space.id)
+        .filter((spaceId: string) => !flaggedSpaces.includes(spaceId)),
+      startDate,
+      endDate
+    )
+  ).filter(proposal => !flaggedProposals.includes(proposal.id));
```

Comment:
let's make it more readable
```suggestion
  const follows = await getFollows(addresses);
  const trustedSpaces = follows
    .map(follow => follow.space.id)
    .filter((spaceId: string) => !flaggedSpaces.includes(spaceId))
  
  const allProposals = await getProposals(trustedSpaces, startDate, endDate);
  const proposals = allProposals
    .filter(proposal => !flaggedProposals.includes(proposal.id));
```

Reply:
Isn't `const follows = await getFollows(addresses);` duplicate ?